config/settings.yml

```diff
@@ -1,4 +1,8 @@
 ---
+:audit_events:
```

Comment:
oohhhh a new number 1 key :)

Reply:
I was going to put it second to :trollface: you. 🤣